db/fixtures/miq_product_features.yml

```diff
@@ -3971,6 +3971,10 @@
       :description: Edit Tags of Middleware Domains
       :feature_type: control
       :identifier: middleware_domain_tag
+    - :name: Stop Middleware Domain
+      :description: Stop Middleware Domain
+      :feature_type: admin
```

Comment:
@h-kataria can you review the features @israel-hdez has linked to above for consistency with other product areas?  Please provide some guidance.  Thx, Dan